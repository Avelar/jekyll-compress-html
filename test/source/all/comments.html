---
layout: compress
---

<div>
  <!-- One -->
  <div>Two</div>

  <!-- Three -->
  <div>Four</div>
  <!--[if IE 8]>Eight<![endif]-->
</div>
