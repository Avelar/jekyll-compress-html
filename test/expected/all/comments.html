<div><div>Two</div><div>Four</div><!--[if IE 8]>Eight<![endif]--></div>
